### PR TITLE
use markdown for sponsor and link list

### PR DESCRIPTION
**Pull request by pagdot**

### CODE REVIEW

**Comment by pagdot** [comment]
dang, just found an issue with '@' signs

**Comment by pagdot** [comment]
ok, only broken for my previewer

File: scrape.py [hunk 1]
```diff
@@ -27,6 +28,14 @@ def get_duration(seconds):
     minutes, seconds = divmod(seconds, 60)
     return f"{minutes} mins {seconds} secs"
 
+def fix_potential_links(entry):
```
Comment (RealOrangeOne) [inline scrape.py:31]: This feels like a lot of extra faff and maintenance, for something [`html2text`](https://pypi.org/project/html2text/) can just do magically, and handle all the edge cases. Someone else's code always wins. I've used the package before, it's really nice for this kind of thing.

**Review by RealOrangeOne — CHANGES_REQUESTED** [review]

File: scrape.py (hunk 1 above)
Comment (pagdot) [reply scrape.py]: Done :)

File: scrape.py [hunk 2]
```diff
@@ -28,6 +30,8 @@ def get_duration(seconds):
     minutes, seconds = divmod(seconds, 60)
     return f"{minutes} mins {seconds} secs"
 
+def format_list(list):
+    return [html2text.html2text(f"{r}") for r in list.contents]
```
Comment (RealOrangeOne) [inline scrape.py:34]: Question: Is it possible to just pass in `str(list)` here? Save iterating, converting to list and back again?

Comment (pagdot) [reply scrape.py]: done :)

File: scrape.py [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import concurrent.futures
 import os
 import shutil
+import re
```
Comment (RealOrangeOne) [inline scrape.py:4]: Looks like you have a stray `re` import left here.
```suggestion
```

**Pull request merged**